qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +132,270 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
+                cals.add_parameter_value(freq, cals.meas_freq, meas)
+
+        cals.update_inst_map_on_add = True
+
+        # Now update the instruction schedule map.
+        cals.update_inst_map()
+
+        return cals
+
+    @property
+    def library(self) -> Optional[BasisGateLibrary]:
+        """Return the name of the library, e.g. for experiment metadata."""
+        return self._library
+
+    @property
+    def default_inst_map(self) -> InstructionScheduleMap:
+        """Return the default and up to date instruction schedule map."""
+        return self._inst_map
+
+    @property
+    def operated_qubits(self) -> Dict[int, List[int]]:
+        """Get a dict describing qubit couplings.
+
+        This is an extension of the coupling map and used as a convenience to help populate
+        the instruction schedule map.
+
+        Returns:
+            A dict where the key is the number of qubits coupled and the value is a list of
+            lists where the sublist shows which qubits are coupled. For example, a three qubit
+            system with a three qubit gate and three two-qubit gates would be represented as
+
+            .. parsed-literal::
+
+                {
+                    1: [[0], [1], [2]],
+                    2: [[0, 1], [1, 2], [2, 1]],
+                    3: [[0, 1, 2]]
+                }
+        """
+
+        # Use the cached map if there is one.
+        if len(self._operated_qubits) != 0:
+            return self._operated_qubits
+
+        # Single qubits
+        for qubit in self._qubits:
+            self._operated_qubits[1].append([qubit])
+
+        # Multi-qubit couplings
+        if self._coupling_map is not None:
+            for coupling in self._coupling_map:
+                self._operated_qubits[len(coupling)].append(coupling)
+
+        return self._operated_qubits
+
+    def get_inst_map(
+        self,
+        group: str = "default",
+        cutoff_date: datetime = None,
+    ) -> InstructionScheduleMap:
+        """Get a new instance of an Instruction schedule map.
+
+        Args:
+            group: The calibration group from which to draw the parameters.
+                If not specified this defaults to the 'default' group.
+            cutoff_date: Retrieve the most recent parameter up until the cutoff date. Parameters
+                generated after the cutoff date will be ignored. If the cutoff_date is None then
+                all parameters are considered. This allows users to discard more recent values that
+                may be erroneous.
+
+        Returns:
+            An instruction schedule map with parameters updated up to the desired cutoff date
+            and from the desired calibration group.
+        """
+        inst_map = InstructionScheduleMap()
+
+        self.update_inst_map(group=group, cutoff_date=cutoff_date, inst_map=inst_map)
+
+        return inst_map
+
+    def update_inst_map(
+        self,
+        schedules: Optional[set] = None,
```

Comment:
```suggestion
        schedules: Optional[Sequence[str]] = None,
```
what is the element of set? this cannot be `Sequence`?

Reply:
You would want to use `Container` rather than `Sequence` if you want to support sets:

```suggestion
        schedules: Optional[Container[str]] = None,
```

Reply:
Let's keep it simple and stick to Sets. I added the type contained in the Set here: f676e9b